src/base/utils/version.h

```diff
@@ -186,10 +186,22 @@ namespace Utils
     };
 
     template <typename T, std::size_t N, std::size_t Mandatory>
-    inline bool operator!=(const Version<T, N, Mandatory> &left, const Version<T, N, Mandatory> &right)
+    constexpr bool operator!=(const Version<T, N, Mandatory> &left, const Version<T, N, Mandatory> &right)
```

Comment:
Why `==`, `<` and `>` operators are class members, but `!=`, `>=` and `<=` are free functions?

Reply:
Yes, I read. Because I asked. Okay, let's keep it that way...